src/linker/Linker/Driver.cs

```diff
@@ -1033,6 +1052,7 @@ static void Usage ()
 			Console.WriteLine ("  --strip-substitutions     Remove XML substitution resources for linked assemblies. Defaults to true");
 			Console.WriteLine ("  --used-attrs-only         Attribute usage is removed if the attribute type is not used. Defaults to false");
 			Console.WriteLine ("  --attribute-defs FILE     Supplementary custom attribute definitions for attributes controlling the linker behavior.");
+			Console.WriteLine ("  --ignore-attributes       Skips reading embedded attributes. Defaults to false");
```

Comment:
I think `--ignore-attributes` is not the right name for this. I actually like the annotation name better but we need something that is be less ambiguous

Reply:
"custom attributes" ? I know it also slightly collides with the terminology in reflection/Cecil, but it's at least a bit more specific.
I personally don't like annotations as the mechanism is all about attributes (the intent is annotations yes, but that's really fuzzy).

Reply:
I think we need to capture that the intent is to control the attributes "placement" not the attributes itself. I'm not sure what most people understand by attributes but considering this is a very advanced feature it should target the people who work in this space.

Reply:
Two points:
* I think we need to synchronize the name of the command line argument with the name of the embedded resource - whatever it is.
* What about "additional attributes", or "additional link attributes"?

Reply:
> I think we need to synchronize the name of the command line argument with the name of the embedded resource 

I agree but `ILLink.Attributes.xml` does not allow you to define attributes only apply existing ones, right?

Reply:
Yes, that's right. That said my understanding of "add attribute" is not to define a new one, but to apply an existing one... but maybe that's just me.

Reply:
Discussed offline with @marek-safar and we sort of agreed on calling it "LinkAttributes". So:
* command line to specify .xml externally `--link-attributes <filename>` (this is to replace the existing `--attribute-defs`)
* embedded file called `ILLink.LinkAttributes.xml`
* command line to enable/disable stripping of the embedded `ILLink.LinkAttributes.xml` will be called `--strip-link-attributes`
* command line to effectively disable the feature `--ignore-link-attributes`

Reply:
We also plan to rename the embedded descriptors to `ILLink.Descriptors.xml` - see https://github.com/mono/linker/issues/1257.